src/helpers/validators/articleValidator.js

```diff
@@ -7,18 +7,12 @@ const schema = joi.object().keys({
     .error(() => errorMessage.title)
     .required(),
   description: joi
-    .string()
-    .required()
-    .error(() => errorMessage.description),
+    .string(),
```

Comment:
I do understand that you removed the required because you allowed null these fields, would you mind keeping the error message as it was @alainburindi ?

Reply:
I was having issues to test them but now it's done